### PR TITLE
Update dependencies of the pub

### PR DESCRIPTION
**Pull request by kanstantsin-bucha**
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] A new third party dependency has been introduced
- [x] Dependencies update 

## **Description**

**_The Problem_**<br/>
Brief description of the problem this PR is attempting to fix

**_Solution_**<br/>
Outline steps taken to fix the stated problem

## **Developer Notes**

I updated dependencies to the latest versions

## **Checklist**

- [x] I have performed a _self-review_ of my own code
- [x] I have verified the impact of my changes on all features of the app
- [x] My code follows the style guidelines of the project
- [x] I have commented on my code, in particular in hard to understand areas
- [x] My changes generate **no new warnings**
- [x] My changes generate **no new errors**

---

## **UI Changes**

Add screenshot/videos here

<details>

<summary> *Title* </summary>

Attach screenshot or video here

</details>

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -4,18 +4,19 @@ version: 0.0.2
...
+  sdk: '>=3.4.1 <4.0.0'
+  flutter: '>=3.24.5'
 
 dependencies:
+  ffi: ^2.1.3
   flutter:
     sdk: flutter
   plugin_platform_interface: ^2.0.2
 
 dev_dependencies:
```
Comment (matteocrippa) [inline pubspec.yaml:16]: we will prob need also:

-   ffigen: 16.0.0

File: pubspec.yaml [hunk 2]
```diff
@@ -4,18 +4,19 @@ version: 0.0.2
 homepage: https://github.com/dotintent/flutter-accessorysetup
 
 environment:
-  sdk: '>=3.4.3 <4.0.0'
-  flutter: '>=3.3.0'
+  sdk: '>=3.4.1 <4.0.0'
+  flutter: '>=3.24.5'
 
 dependencies:
+  ffi: ^2.1.3
```
Comment (matteocrippa) [inline pubspec.yaml:11]: And this one:

  objective_c: 4.0.0

**Review by matteocrippa — APPROVED** [review]
Looks ok, I've just add a not about two extra missing libs

File: pubspec.yaml (hunk 1 above)
Comment (kanstantsin-bucha) [reply pubspec.yaml:16]: Thanks, I will add them in the next Pr

File: pubspec.yaml (hunk 2 above)
Comment (kanstantsin-bucha) [reply pubspec.yaml:11]: Thanks, I will add them in the next Pr

**Pull request merged**